xbmc/video/VideoDatabase.cpp

```diff
@@ -193,6 +193,13 @@ void CVideoDatabase::CreateTables()
 
   CLog::Log(LOGINFO, "create uniqueid table");
   m_pDS->exec("CREATE TABLE uniqueid (uniqueid_id INTEGER PRIMARY KEY, media_id INTEGER, media_type TEXT, value TEXT, type TEXT)");
+
+  CLog::Log(LOGINFO, "create type table");
+  m_pDS->exec("CREATE TABLE type (type_id INTEGER PRIMARY KEY, name TEXT)");
+  m_pDS->exec("INSERT INTO type VALUES(1, 'STANDARD')");
```

Comment:
Yes, this will be in a dialog to let user choose. I'll change this to capitalized version. I personally like all this kind of types in uppercase, like REMASTERED/EXTENDED/LIIMITED, etc. BTW, I want to add the common ones (about 30) to the DB, so user no need to add them manually, but haven't figured out a good place to do this? Can you give me some light on this? :)